PureBasic.sublime-syntax

```diff
@@ -415,3 +394,35 @@ contexts:
       captures:
         0: variable.other.constant.purebasic
         1: punctuation.definition.variable.purebasic
+
+  operators_all:
```

Comment:
I broke the pattern myself with the `constants` context as I see now. In case of an amendment, can you move this top level context and `constants` to the alphabetically correct position in the context list?